controllers/common/types.go

```diff
@@ -49,14 +49,20 @@ func NewReconciler(reconcile reconciler.InnerReconciler, c client.Client, log lo
 func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	var err error
 
-	r.Log.Info("reconciling a common chaos", "name", req.Name, "namespace", req.Namespace)
+	r.Log.Info("Reconciling a common chaos", "name", req.Name, "namespace", req.Namespace)
 	ctx := context.Background()
 
 	chaos := r.Object()
 	if err = r.Get(ctx, req.NamespacedName, chaos); err != nil {
 		r.Log.Error(err, "unable to get chaos")
 		return ctrl.Result{}, err
 	}
+
+	status := chaos.GetStatus()
+	if status == nil {
+		status = &v1alpha1.ChaosStatus{}
```

Comment:
`GetStatus` returns a reference from inner of chaos object. It will never return `nil`.

If you give it a newly created status, set/get value of it will have no effect.

Reply:
my fault, I will fix it

Reply:
@YangKeao I have fixed it, PTAL again, Thanks!